manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -0,0 +1,686 @@
+---
+# Source: istiod-remote/templates/istiod-injector-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-sidecar-injector
+  namespace: default
+  labels:
+    istio.io/rev: default
+    release: istiod-remote
+data:
+
+  values: |-
+    {
+      "global": {
+        "arch": {
+          "amd64": 2,
```

Comment:
a note for other reviewers, at some point arm enablement must be done for the arch selection - if this field is currently used.